### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA announces additional inappropriate information when "Create Authentication Files" dialog   is opened 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1477]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3.  In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. "Create Authentication Files" dialog will open.
6. Observe the behaviour of Jaws and NVDA 

**Actual Result:**
Jaws and NVDA announces additional inappropriate information when "Create Authentication Files" dialog   is opened 

**Jaws behaviour:**
 untitled4 - IntelliJ IDEA frame
Create Authentication Files dialog
Subscription name Subscription id C:\Users\abby Destination folder: A new Active Directory service principal representing this IDE will be created as needed and as allowed by your access permissions.
The service principal will be granted Contributor-level access to each selected subscription. Select the subscription(s) to create credentials for: 
Start Button
To activate press spacebar

**NVDA Behaviour:**
Create Authentication Files  dialog  Destination folder:
Select the subscription(s) to create credentials for:
Start  button

**Expected Result:**
Jaws and NVDA should announce only the dialog name when dialog is opened .
i.e Create Authentication Files dialog

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![cretae authenctication-nvda](https://user-images.githubusercontent.com/22956130/36104865-2eab2942-1039-11e8-8d6c-5a7566d95db5.jpg)
![create authntication files](https://user-images.githubusercontent.com/22956130/36104872-32024fc6-1039-11e8-85d5-c9229dc4210e.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749914
